tsconfig.json

```diff
@@ -21,9 +22,11 @@
 		],
 		"baseUrl": ".",
 		"paths": {
-			"@utils/*": ["app/utils/*"],
-			"@api/*": ["app/api/*"]
-		}
+			"#utils/*": ["./app/utils/*"],
```

Comment:
These really should not be needed, but during next.js build them seem to ignore Node import path aliases... 😢